octree/include/pcl/octree/impl/octree2buf_base.hpp

```diff
@@ -39,6 +39,8 @@
 #ifndef PCL_OCTREE_2BUF_BASE_HPP
 #define PCL_OCTREE_2BUF_BASE_HPP
 
+#include <pcl/octree/octree2buf_base.h>
```

Comment:
Is this really necessary? I normally only see the corresponding  *.hpp being included at the end of the *.h file and the *.hpp never includes the corresponding *.h .